src/dotnet/Chat.UI.Blazor/Components/LeftPanel/ThinLeftPanel.razor

```diff
@@ -8,7 +9,7 @@
         </div>
         <div class="flex-y w-full grow items-center gap-y-3 md:gap-y-2">
             <ButtonSquare
-                Click="OnCreateButtonClick"
+                Click="OnCreateNewChat"
```

Comment:
as discussed offline, we stick to On-Component-Action convention
so it would be something like `OnCreateChatClick` or `OnCreateChatButtonClick`

Reply:
Done